static/bidder-info/resetdigital.yaml

```diff
@@ -0,0 +1,21 @@
+adapters:
+  resetdigital:
+    endpoint: http://b-us-east14.resetdigital.co:9001
```

Comment:
@bruno-siira is it possible to use standard http or https port i.e `http(s)://b-us-east14.resetdigital.co` instead of `9001`

Reply:
If necessary yes, but its a practice for this kind of our servers. It's needed for the approval?

Reply:
We have a situation where the only server that we have that supports the Forced Bid is not with the standard port. We have now server with that standard port but doesn't allow the forced bidder so it can't pass the auto test. Any idea to solve this @onkarvhanumante ?

Reply:
> We have a situation where the only server that we have that supports the Forced Bid is not with the standard port. We have now server with that standard port but doesn't allow the forced bidder so it can't pass the auto test. Any idea to solve this @onkarvhanumante ?

I am not sure how to resolve it